fixtures/case_entity.yaml

```diff
@@ -970,3 +970,25 @@ App\Entity\ResidentComplaintBoardCase:
     municipality: '@municipality-randers'
     currentPlace: Oprettet
     assignedTo: '@user-caseworker'
+
+App\Entity\FenceReviewCase:
+  test-case-fence-aarhus-1:
+    accused: Anders And
+    accusedCPR: 0101010101
+    complainant: Jens Jensen
+    complainantCPR: 1234567890
+    complainantAddress: '@test-case-aarhus-1-complainant-address'
+    accusedAddress: '@test-case-aarhus-1-lease-address'
+    finishHearingDeadline: <(new \DateTime('2022-05-20'))>
+    finishProcessingDeadline: <(new \DateTime('2022-07-11'))>
+    complaintCategory: '@fence-complaint-aarhus'
+    caseNumber: 21-100001
```

Comment:
Maybe i worded this wrong when you asked, but `caseNumber` is supposed to be unique no matter board and municipality. 

I do realise this is unique compared to the others case numbers, but consider changing it to be increasing (21-000034).

Reply:
Fixed.